Deploy error  due to the undefined 'blockPublicAccess' property

This issue is to address the error below:

When merging the changes to main branch, following deploy error would be triggered:

```
Run npx aws-cdk synth
  npx aws-cdk synth
  shell: /usr/bin/bash -e {0}
  env:
    LBL_STAGE: production
    DEBIAN_FRONTEND: noninteractive
    TEMPLATES_FILE_NAME: templates.json
    AWS_DEFAULT_REGION: us-east-2
    AWS_REGION: us-east-2
    AWS_ACCESS_KEY_ID: ***
    AWS_SECRET_ACCESS_KEY: ***
/home/runner/work/ctrl-flow-dev/ctrl-flow-dev/cdk/lib/cdk-stack.ts:22
    const bucket = new Bucket(this, `lbl-client-${stage}`, {
                   ^
ValidationError: Cannot use 'publicReadAccess' property on a bucket without allowing bucket-level public access through 'blockPublicAccess' property.
    at path [LblCdkStackproduction/lbl-client-production] in aws-cdk-lib.aws_s3.Bucket

    at new LblCdkStack (/home/runner/work/ctrl-flow-dev/ctrl-flow-dev/cdk/lib/cdk-stack.ts:22:20)
    at Object.<anonymous> (/home/runner/work/ctrl-flow-dev/ctrl-flow-dev/cdk/bin/cdk.ts:9:1)
    at Module._compile (node:internal/modules/cjs/loader:1529:14)
    at Module.m._compile (/home/runner/work/ctrl-flow-dev/ctrl-flow-dev/cdk/node_modules/ts-node/src/index.ts:1618:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)
    at Object.require.extensions.<computed> [as .ts] (/home/runner/work/ctrl-flow-dev/ctrl-flow-dev/cdk/node_modules/ts-node/src/index.ts:1621:12)
    at Module.load (node:internal/modules/cjs/loader:1275:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1096:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)
    at phase4 (/home/runner/work/ctrl-flow-dev/ctrl-flow-dev/cdk/node_modules/ts-node/src/bin.ts:649:14)
Subprocess exited with error 1
Error: Process completed with exit code 1.
```

--- a/cdk/lib/cdk-stack.ts
+++ b/cdk/lib/cdk-stack.ts
@@ -22,6 +22,12 @@ export class LblCdkStack extends cdk.Stack {
     const bucket = new Bucket(this, `lbl-client-${stage}`, {
       bucketName: `lbl-client-${stage}`,
       publicReadAccess: true,
+      blockPublicAccess: {
+        blockPublicAcls: false,
+        blockPublicPolicy: false,
+        ignorePublicAcls: false,
+        restrictPublicBuckets: false,
+      },
       websiteIndexDocument: "index.html",
       websiteErrorDocument: "index.html",
       autoDeleteObjects: true,

